Array_Functions/Lower_Bound_Binary_Search/lower_bound_bs.cpp

```diff
@@ -0,0 +1,25 @@
+# include <iostream>
+# include <vector>
+using namespace std;
+
+int lower_bound_bs(vector<int> arr, int target){
+    int beg = 0;
+    int end = arr.size()-1;
+
+    while(beg<=end){
+        int mid = (beg+end)/2;
```

Comment:
never use beg+end, as the values may overflow

Reply:
Relevant changes has been made according to your instructions, kindly review them.